ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-ref.yaml

```diff
@@ -72,6 +72,9 @@ ref:
       \[sig-network\] DNS should resolve DNS of partial qualified names for services
       \[sig-network\] DNS should resolve DNS of partial qualified names for the cluster
       \[sig-network\]\[endpoints\] admission TestEndpointAdmission
+      \[sig-network\] \[Feature:IPv6DualStack\] \[LinuxOnly\] should have ipv4 and ipv6 node podCIDRs
+      \[sig-network\] \[Feature:IPv6DualStack\] \[LinuxOnly\] Granular Checks: Services Secondary IP Family should function for client IP based session affinity: http
+      \[sig-network\] \[Feature:IPv6DualStack\] \[LinuxOnly\] Granular Checks: Services Secondary IP Family should function for client IP based session affinity: udp
```

Comment:
Is there a bug for this? I don't think it's a good idea to disable tests that are meant to pass on dualstack (these are explicitly marked Feature:IPv6DualStack) -- that are now failing.

Reply:
This is interim solution to make dualstack jobs green, the first thing I'm going to do is to enable these tests on dualstack.

Reply:
or if we can wait for a while, maybe we can focus on fixing this problem. But until that time dualstack will be red.

Reply:
It seems like these tests are networking related, I'd file a bug and hand it off to the OVN team to investigate. IMHO we shouldn't be using TEST_SKIPS in baremetal anymore, the goal is to remove everything from there. If we really have to skip these dualstack tests, the skip should go in openshift/origin and marked `[Disabled:Broken]` with a bug attached.

https://github.com/openshift/origin/blob/master/test/extended/util/annotate/rules.go#L32

Reply:
Thanks for the detailed explanation. I'll follow the steps you suggested and closing this PR.